refarch-integrations/refarch-dms-integration/refarch-dms-integration-core/src/test/java/de/muenchen/refarch/integration/dms/adapter/out/fabasoft/DMSErrorHandlerTest.java

```diff
@@ -50,9 +50,10 @@ void handleDmsException() {
     }
 
     @Test
+    @SuppressWarnings("PMD.UnitTestShouldIncludeAssert")
     void uebertragungErfolgreich() throws DmsException {
```

Comment:
We should rename the method (to have a english name). I think this would fit best in a separate issue if more occurences are found.

Reply:
Think a separate issue isn't needed as i replaced every german method name if appropriate.
Fixed that one